back-end/src/dms/admin.py

```diff
@@ -5,6 +5,8 @@
     Book,
     FavoriteBook,
 )
+from dms.models.documents import (
+    File,)
```

Comment:
I don't really like how it looks, but formatter says it should look like this.
And is it necessary here?